### PR TITLE
Security functionality added

### PR DESCRIPTION
**Pull request by Vegasq**
- Google reCaptcha on add page
- Anonymouse reports
- (IMPORTANT) this fix changes database shema, and i've discovered
  that you do not use migrations. So i've created basic migration,
  and one abowe it. To use it from now you should call:
   ./manage.py migrate --fake-initial # fakly apply your structure
   ./manage.py migrate # apply change

### CODE REVIEW

File: corruption_tracker/settings.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# This information should be obtained by Google
```
Comment (autogestion) [inline corruption_tracker/settings.py]: better to move it in local settings

File: corruption_tracker/settings.py [hunk 2]
```diff
@@ -16,6 +16,9 @@
...
+RECAPTCHA_SECRET = ""
```
Comment (autogestion) [inline corruption_tracker/settings.py]: better to move it in local settings

File: claim/views.py [hunk 3]
```diff
@@ -36,17 +51,31 @@ def get_claims(request, polygon_id):
 
 @login_required
 def add_claim(request):
+    if not request.POST.get('g-recaptcha-response', False):
```
Comment (Dovbo) [inline claim/views.py]: Let's make 3 types of users: anonymous, accounted and de-anonymized. Thus, anonymous will get capcha in 100% of claims, accounted user - 20% and de-anonymized 0%. Capcha in this model is spam protection, accounts+capcha is idiot protection. Also such differentiation should stimulate users to create accounts and\or intensify civil activity.

Comment (Vegasq) [inline claim/views.py]: Don't get it. Please create wiki page with description. So far i made:
- You can disable recaptcha from config
- If user is logged in - do not ask captcha
- If user not authorized - ask capthca

**Pull request merged**